File: Examples/PHP/managing-recognition/without-cloud-storage/read-barcode-from-external-image-url.php
<?php

/**
 * Include the SDK by using the autoloader from Composer.
 */
require __DIR__ . '/../../vendor/autoload.php';

/**
 * Include the configuration values.
 *
 * Ensure that you have edited the configuration.php file
 * to include your application keys.
 */
$config = require __DIR__ . '/../../configuration.php';

$apiKey = $config ['apiKey'];
$appSid = $config ['appSid'];
$out_folder = $config ['outFolder'];
$data_folder = '../../data/'; // resouece data folder

/**
 * The namespaces provided by the SDK.
 */
use Aspose\Barcode\BarcodeApi;
use Aspose\Barcode\APIClient;

// ExStart:1

\Aspose\Barcode\AsposeApp::$apiKey = $apiKey;
\Aspose\Barcode\AsposeApp::$appSID = $appSid;
;

// Instantiate Aspose BarCode Cloud API SDK
$barcodeApi = new BarcodeApi ();

// Set the image file url
$url = "http://www.barcoding.com/images/Barcodes/code93.gif";

// The barcode type.
// If this parameter is empty, autodetection of all supported types is used.
$type = "";

// Set mode for checksum validation during recognition
$checksumValidation = "";

// Set if FNC symbol stripping should be performed.
$stripFnc = "False";

// Set recognition of rotated barcode
$rotationAngle = 0;

// Set exact number of barcodes to recognize
$barcodesCount = 0;

// Set recognition of barcode inside specified Rectangle region
$rectX = "";
$rectY = "";
$rectWidth = "";
$rectHeight = "";

// Set 3rd party cloud storage server (if any)
$storage = "";

// Set folder location at cloud storage
$folder = "";

// Set local file (if any)
$file = "empty.fle";

try {
	
	// invoke Aspose.BarCode Cloud SDK API to read barcode from external image URL
	$response = $barcodeApi->PostBarcodeRecognizeFromUrlorContent ( $type, $checksumValidation, $stripFnc, $rotationAngle, $url, $file );
	
	if ($response != null && $response->Status = "OK") {
		print_r ( $response );
	}
} catch ( \Aspose\Barcode\ApiException $exp ) {
	echo "Exception:" . $exp->getMessage ();
}
//ExEnd:1

File: Examples/Ruby/managing-recognition/without-cloud-storage/read-barcode-from-external-image-url.rb
# Require asposecloudsdk

require 'aspose_barcode_cloud'

include AsposeStorageCloud
include AsposeBarcodeCloud
 
configFile = File.expand_path("../../../data/config.json", __FILE__);
configProps = JSON.parse( IO.read(configFile, encoding: 'utf-8') )
data_path = File.expand_path("../../../data/", __FILE__);
out_path = configProps['out_folder'];
  
#ExStart:1
AsposeApp.app_key_and_sid(configProps['api_key'], configProps['app_sid'])
AsposeBarcodeCloud::Configuration.debugging = false
AsposeStorageCloud::Configuration.temp_folder_path = out_path

#Instantiate Aspose.BarCode API SDK
@barcodeApi = BarcodeApi.new

#Set the image file url
url = 'http://www.barcoding.com/images/Barcodes/code93.gif';

opts = {url: url}

#Invoke Aspose.BarCode Cloud SDK API to read barcode from external image URL                                   
response = @barcodeApi.post_barcode_recognize_from_urlor_content(opts)

#print response
response.barcodes.each { |barcode| puts  barcode }

#ExEnd:1